06_modules_lab/01.py

```diff
@@ -1,4 +1,15 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+##################Part 1
+import os
+import sys
+
+if len(sys.argv) != 2:
+    print('Usage: %s <number of repetions>' % sys.argv[0])
+    sys.exit(1)
+
+(_, num) = sys.argv
+print('number of repetitions = {}'.format(num) )
+#print('{}'.format(type(num)))
+
+for i in range (0, int(num)):
+    print('Hello Python')
+

```

Comment:
👍 
Minor issues:
* You don't need to import ```os``` here
* Better to always use ```format``` and not ```%```.
* Better to write ```num = sys.argv[1]```